static/js/stat_var_hierarchy/stat_var_search.tsx

```diff
@@ -236,23 +252,56 @@ export class StatVarHierarchySearch extends React.Component<
     });
   };
 
+  // Creates a jsx element with parts of the string matching a set of words
+  // highlighted.
+  // eg. s: "test string abc def", matches: ["abc", "blank"]
+  //     would return s with "abc" highlighted
+  private getHighlightedJSX(
```

Comment:
this is nice - pretty subtle as well, and would be nice to have a basic test for it.

Reply:
added test